### PR TITLE
Enable self-service invoicing

### PR DESCRIPTION
**Pull request by woeltjen**
## Description
Make BEE payments full self-service. Also removes ETH/DAI payments, as we don't have a functional refund flow for those payment types.

## How to Test
1. Make a BEE booking (on a listing with a host wallet address)
2. Wait for approval to complete (note that this happens asynchronously; app will move on)
3. Go to Host portal
4. Approve the booking
5. Wait for transaction to complete
6. Run the payments event handler on the backend (see README in `packages/PaymentsEventHandler` in server)
7. Expect booking to show as Approved

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

### CODE REVIEW

**Comment by woeltjen** [comment]
This is ready for test now. Want to deploy to mainnet and update addresses here before merging, but want to test on testnet before deploying on mainnet. (Have completed self-testing for booking & invoice in nominal conditions)

**Comment by kcvan** [comment]
getting stuck on part 4 with the error:
 `GraphQL error: Payment not found for bookingId 5c805d31311f128796dd3046`

running this branch and the backend one

**Comment by woeltjen** [comment]
> 5c805d31311f128796dd3046

When I look at dev dbee, this booking has currency set to USD. It was also created 2019-03-06 23:52:17

I did see some emails come through for a BEE booking with ID of 5c9bbdc878736a77e2ad2ed2

**Comment by kcvan** [comment]
huh strange...let me try again

**Comment by kcvan** [comment]
Tried again with Jeremy, for booking id `5c9bc3d3173332af9ca1a347` and my booking `5c9bbdc878736a77e2ad2ed2`.

getting:
```
ALERT: Transaction Error. Exception thrown in contract code.
```
Screen Shot 2019-03-27 at 11 47 22 AM

**Comment by woeltjen** [comment]
The contract address (0x3a...) in that screenshot looks outdated relative to this PR

image

 May need to kill/restart the dev server to pick up on changes in settings?

**Comment by kcvan** [comment]
good catch on the server restart!  Tried making a new booking and running into the same error i think (but with the new address)
Screen Shot 2019-03-27 at 11 58 59 AM
